.travis.yml

```diff
@@ -10,13 +10,17 @@ before_script:
   # ensure we only have declared dependencies
   - npm prune
   # fetch certificates for integration tests against foundation-dev pod
-  - "curl -s https://raw.githubusercontent.com/symphonyoss/contrib-toolbox/master/scripts/download-files.sh | bash"
+  - "if [[ $TRAVIS_PULL_REQUEST -eq 'false' ]]; then curl -s https://raw.githubusercontent.com/symphonyoss/contrib-toolbox/master/scripts/download-files.sh | bash; fi;"
 
 script:
+  # Fail if license violations are found
+  - npm run-script license-violations
+  # Generate a list of dependencies that require a notice, as per license
+  - npm run-script license-notice-warns
```

Comment:
Not sure why we're warning on Unlicense -- it's Category A. Public Domain is vague enough that we ought to warn.